README.md

```diff
@@ -26,6 +26,7 @@ addon | version | summary
 [stock_available_immediately](stock_available_immediately/) | 9.0.1.1.0 | Ignore planned receptions in quantity available to promise
 [stock_available_mrp](stock_available_mrp/) | 9.0.1.0.0 | Consider the production potential is available to promise
 [stock_partner_lot](stock_partner_lot/) | 9.0.1.0.0 | Show lots on the partners that own them
+[stock_reserve](stock_reserve/) | 9.0.0.2.0 | Stock reservations on products
```

Comment:
This file is maintained automatically & editing it as such will cause conflicts with our automation. Please reverse change